01-kogito-travel-agency/src/main/resources/org/acme/travels/hotelBooking.bpmn2

```diff
@@ -1,109 +1,128 @@
     </bpmn2:endEvent>
-    <bpmn2:sequenceFlow id="SequenceFlow_1" tns:priority="1" sourceRef="ServiceTask_1" targetRef="EndEvent_1"/>
-    <bpmn2:sequenceFlow id="SequenceFlow_2" tns:priority="1" sourceRef="StartEvent_1" targetRef="ServiceTask_1"/>
   </bpmn2:process>
-  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
-    <bpmndi:BPMNPlane id="BPMNPlane_Process_1" bpmnElement="hotelBooking">
-      <bpmndi:BPMNShape id="BPMNShape_StartEvent_1" bpmnElement="StartEvent_1">
-        <dc:Bounds height="36.0" width="36.0" x="60.0" y="72.0"/>
-        <bpmndi:BPMNLabel>
-          <dc:Bounds height="11.0" width="52.0" x="52.0" y="108.0"/>
-        </bpmndi:BPMNLabel>
+  <bpmndi:BPMNDiagram>
+    <bpmndi:BPMNPlane bpmnElement="hotelBooking">
+      <bpmndi:BPMNShape id="shape_EndEvent_1" bpmnElement="EndEvent_1">
+        <dc:Bounds height="56" width="56" x="475" y="66"/>
       </bpmndi:BPMNShape>
-      <bpmndi:BPMNShape id="BPMNShape_ServiceTask_1" bpmnElement="ServiceTask_1" isExpanded="true">
-        <dc:Bounds height="50.0" width="110.0" x="190.0" y="65.0"/>
-        <bpmndi:BPMNLabel>
-          <dc:Bounds height="11.0" width="43.0" x="223.0" y="84.0"/>
-        </bpmndi:BPMNLabel>
+      <bpmndi:BPMNShape id="shape_ServiceTask_1" bpmnElement="ServiceTask_1">
+        <dc:Bounds height="60" width="129" x="249" y="66"/>
       </bpmndi:BPMNShape>
-      <bpmndi:BPMNShape id="BPMNShape_EndEvent_1" bpmnElement="EndEvent_1">
-        <dc:Bounds height="36.0" width="36.0" x="368.0" y="72.0"/>
-        <bpmndi:BPMNLabel>
-          <dc:Bounds height="11.0" width="50.0" x="361.0" y="108.0"/>
-        </bpmndi:BPMNLabel>
+      <bpmndi:BPMNShape id="shape_StartEvent_1" bpmnElement="StartEvent_1">
+        <dc:Bounds height="56" width="56" x="110" y="68"/>
       </bpmndi:BPMNShape>
-      <bpmndi:BPMNEdge id="BPMNEdge_SequenceFlow_1" bpmnElement="SequenceFlow_1" sourceElement="BPMNShape_ServiceTask_1" targetElement="BPMNShape_EndEvent_1">
-        <di:waypoint xsi:type="dc:Point" x="300.0" y="90.0"/>
-        <di:waypoint xsi:type="dc:Point" x="334.0" y="90.0"/>
-        <di:waypoint xsi:type="dc:Point" x="368.0" y="90.0"/>
-        <bpmndi:BPMNLabel/>
+      <bpmndi:BPMNEdge id="edge_shape_ServiceTask_1_to_shape_EndEvent_1" bpmnElement="SequenceFlow_1">
+        <di:waypoint x="378" y="96"/>
+        <di:waypoint x="378" y="94"/>
+        <di:waypoint x="475" y="84"/>
       </bpmndi:BPMNEdge>
-      <bpmndi:BPMNEdge id="BPMNEdge_SequenceFlow_2" bpmnElement="SequenceFlow_2" sourceElement="BPMNShape_StartEvent_1" targetElement="BPMNShape_ServiceTask_1">
-        <di:waypoint xsi:type="dc:Point" x="96.0" y="90.0"/>
-        <di:waypoint xsi:type="dc:Point" x="143.0" y="90.0"/>
-        <di:waypoint xsi:type="dc:Point" x="190.0" y="90.0"/>
-        <bpmndi:BPMNLabel/>
+      <bpmndi:BPMNEdge id="edge_shape_StartEvent_1_to_shape_ServiceTask_1" bpmnElement="SequenceFlow_2">
+        <di:waypoint x="146" y="86"/>
+        <di:waypoint x="249" y="96"/>
```

Comment:
Hey @inodeman 

I'm pointing out only these waypoints here, but this applies to ALL proceses below as well.

Looking at the "old" vs "new" waypoints I see there are still 3 points for each line, as before, and in this case if you notice both waypoints are at `[249, 96]`, so 2 of the 3 waypoints are the same one. 

So this results on the process looking good in the UI, at first look:

![points0](https://user-images.githubusercontent.com/4602417/80264223-bc8ea180-8693-11ea-8a56-0b84bba1f837.png)

But in fact, as there are still 3 points, when you move the end waypoint from the UI, it results on this:

![points2](https://user-images.githubusercontent.com/4602417/80264095-33776a80-8693-11ea-990d-cd5c50df0be2.png)

So in general, I expect refactoring those lines in that processes by moving from 3 to 2 point-based lines. Otherwise, once moving the "duplicated" concrete waypoint from the UI, the user will notice that a new point is there but not sure if just created and was already there... 

Does it makes sense?

Reply:
Hi @romartin makes sense, I just did not want to re create the connectors in case there was something needed from the old diagram, let me know if there is a way to just delete a point. Will check with you Monday morning

Reply:
you can delete a waypoint just by doublie clicking on it in the UI. Also you can just update the xml manually and remove the waypoint where appears.